website/content/docs/connect/transparent-proxy.mdx

```diff
@@ -1,13 +1,11 @@
 ---
 layout: docs
-page_title: Connect - Transparent Proxy
-sidebar_title: Transparent Proxy
-description: |-
-  Transparent proxy is used to direct inbound and outbound traffic to services via the Envoy proxy and configure
-  upstreams via intentions.
+page_title: Transparent Proxy | Service Mesh
+description: >-
+Learn how transparent proxies enable Consul on Kubernetes to direct inbound and outbound traffic through the service mesh. Use a transparent proxy to increase application security without configuring individual services and intentions.
 ---
 
-# Transparent Proxy
+# Transparent Proxies in a Service Mesh
```

Comment:
Transparent Proxy is a mode/feature, not a type of proxy. So this should be "Transparent Proxying in a service mesh"

Reply:
```suggestion
# Transparent Proxying in a Service Mesh
```